doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,16 +456,73 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
+
+Key rotation for the redo log has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo").
+
+The changed page tracking may read pages which have the encryption bit set.  The redo log does not have the encryption key and decryption fails and generates an error message. A flag lets the read process safely ignore encryption errors in this case.
```

Comment:
I don't think this is true? If this is about PS-5541, that doesn't do anything with the redo log - it's only an internal "hack" to read pages like logpages.

Reply:
This question is still there: if this block is about PS-5541, it shouldn't be here at all - that bug doesn't have to do anything with the redo logs really.

Reply:
Removed.

Reply:
It's still here? Or you didn't push the latest changes?